dev-tex/rail/rail-1.2_p1-r2.ebuild

```diff
@@ -18,11 +18,17 @@ BDEPEND="app-arch/unzip
 	app-alternatives/yacc
 	app-alternatives/lex"
 
+PATCHES=( "${FILESDIR}/${P}-C23.patch" )
+
 src_compile() {
-	emake -j1 \
+	emake \
 		CC="$(tc-getCC)" \
 		CFLAGS="-DYYDEBUG ${CFLAGS} ${LDFLAGS}" \
-		rail rail.dvi
+		rail
+}
+
+src_test() {
```

Comment:
Hm, why is this in src_test?

Reply:
It runs itself on its own documentation. That then remains uninstalled.

I'll double-check.

Reply:
OK, put a comment, I think. It just looked odd to see it move from `src_compile`. I assume it has no other tests.

Reply:
I was wrong. documentation generated by rail.dvi target is then put into installed package.

Thank you for making me double-check.